apps/company/views.py

```
@@ -122,6 +171,689 @@ class CompanyStocksView(APIView):
+            region_obj = supplier.values_list("warehouse",flat=True)
+            context = {"market": service, "region_obj": region_obj}
+        else:
+            context = {"market": service}
+        
+        paginator = Paginator(supplier, per_page=page_size)
+        page = paginator.get_page(page)
+        count = paginator.count
+        serializer = RecomamandationSupplierSerializer(page, context=context, many=True)
+        
+        if sort and sort in ["-1", "1"]:
+            ordering_by_quantity = False if sort =="-1" else True
```

Ruff

Use `not ...` instead of `False if ... else True`

Replace with `not ...`

(SIM211)